resources/assets/components/CampaignsTable.js

```diff
@@ -97,13 +63,6 @@ const CampaignsTable = ({ isOpen, filter }) => {
     });
   };
 
-  // If we're filtering results & can load more, do so automatically:
-  useEffect(() => {
-    if (filter !== '' && !loading && hasNextPage) {
-      handleViewMore();
-    }
-  }, [filter, endCursor]);
-
```

Comment:
Since this was really a "hack" to deal with the fact that we were loading _all_ campaigns and searching client-side, I removed this (since it feels relatively unlikely for folks to search & not find what they're looking for on the first page).